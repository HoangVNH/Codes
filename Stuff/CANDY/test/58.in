10
83
DVVVVDDDDVVDVXVDVDDVDXXXDXDVVVDXDDXVDVVDXVVXXVVDVXXVVXVXDVVXXXDDDDXDVXVDVVXVDDVVDDX
68
VVXDXVVXXVXDVXDXVDDXDVVXVVXDDXVDDDDXXVXDVXVVXXDDVDXDXDXVVXDXXXDDXVDD
90
XDXXXDDXDVVDDDDDDVXDXXXXXXDDXDDVXVVVDXVDXVVDDXXXVXXVDVVXVXDDDXXDDVDDVVVDVVVVXXXVVDDDVXDDVD
91
DXXDXXXVVDXXDVXDDXVDDXDDXXVVVXDVVDDDXDXVXDXVDXVVDXXVVXVXDXXVDXXDXVDVXXDVDVXXXDVVDXDDDVDXVVX
83
VDXVVVVXVDVVDDVXXDDDVDVDVDXVXVXDXXDXDXXVVDVVDDDVXXVDDDVXVVDDXVXDDVDDXDDVVVVXXVXXVDX
74
XXXXDXXXXDXXXDDXXDDXXVVDVVXVXDDXDDVVXVXXDDXXVXVXXVDDXDDDDDDDDVVXXDXDVXVXDD
59
DDXDDXDXVVVVVVDXXDVDVDVVXXXXVDVVDDDXDDVXDVDXDDDXVDXDVDDVVVV
71
VVVXDDVVDDVXVXVVVVDDDVVDXDVDVDXDVVDVXDVDVDVDXVDDDDVVXVDXVXDXDXXXXVVXVXV
55
VDXXDDXVXDDVXVDVVVVDXDDXXVXVVDVXDVDDVDDVXDDDXVXXDXXVXXD
52
VXXDDVDXXXDXVXDXVDVVXVXVVDXXXVXDVVVVXVDVVVDVDVVVXDVV
