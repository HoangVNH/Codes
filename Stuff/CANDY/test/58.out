81
67
89
90
81
73
58
70
54
50
